.env

```
@@ -0,0 +1,2 @@
+GOCARDLESS_SECRET_ID=554412b2-22c2-4362-b7bb-5516ff718a91
+GOCARDLESS_SECRET_KEY=800b3549d4550c68f4d5fd156d458a0f9609e839a945a422d882d910ed8776db14cd62325ebd7ceabf2e8588881a40107336dfd30dacf3c8038f799787dba1a4
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)